### ISSUE TITLE
STACER blank screen with recent versions .. 

### ISSUE THREAD

**Issue by netmediasurfer** [oguzhaninan/Stacer#56]
1. Unable to get the last few versions of Stacer to work on Ubuntu 14.04.5. All fail to load anything on the screen. I get a blank grey screen as someone previously commented.  I think the only one that I was able to get to work was version 1.0.4.

2. Stacer 1.0.5 & 1.0.6 fail to create a start menu entries within Accessories in Ubuntu 14.04.5. 

Can you make it so that Stacer has an option to disable the update available notification prompt. It's annoying to not be able to turn that off in Stacer 1.0.4.

Thanks!

**Comment by oguzhaninan** [comment]
Hi,
That's the reason for your problem. https://github.com/oguzhaninan/Stacer/issues/55#issuecomment-308251972 https://github.com/oguzhaninan/Stacer/issues/44#issuecomment-298310770 
I will soon release an update that will fix this problem.

**Comment by netmediasurfer** [comment]
Hi, thanks for the clue. I got it working by installing "systemd" to Ubuntu 14.04.5. I was able to launch all of them afterwards without the blank screen issue I was experiencing.

sudo apt-get install systemd -f 

I run Stacer 1.7 by running the command "stacer" from start menu command terminal entry box. It didn't install a start menu entry for me within accessories like the previous versions.

Just one more thing ..

I Go to "System Cleaner" & select any option there. Run the "System Scan" & click "Select All" once the results display on the right. Click "Clean" & Stacer prompts me that it failed. However, if I close & open Stacer again & check for things to clean it actually cleaned them out successfully.

May want to look into that "failed cleaning" notification.

Thanks!
